appmonitoring/ts/src/CertificateManager.ts

```diff
@@ -186,14 +202,28 @@ export class CertificateManager {
                 throw new Error("MutatingWebhookConfiguration not found or is malformed!");
             }
 
-            return Buffer.from(mutatingWebhookObject.webhooks[0].clientConfig.caBundle, 'base64').toString('utf-8');
+            let caBundle: string = null;
+
+            try {
```

Comment:
This is very confusing, why is this try nested into the first try? Let's have two separate tries in sequence - one for the API call, one for interpreting its results. As-is is very confusing.